templates/base/head.tmpl

```diff
@@ -69,7 +69,6 @@
 			Minicolors: {{if .RequireMinicolors}}true{{else}}false{{end}},
 			SimpleMDE: {{if .RequireSimpleMDE}}true{{else}}false{{end}},
 			Tribute: {{if .RequireTribute}}true{{else}}false{{end}},
-			U2F: {{if .RequireU2F}}true{{else}}false{{end}},
```

Comment:
Unnecessary, handled by https://github.com/go-gitea/gitea/pull/11585 and will cause conflict for zero benefit